### ISSUE TITLE
When using the AG-Grid theme focused headers use a black outline

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#1001]
### Package name(s)

AG Grid Theme v1.0.0 (@salt-ds/ag-grid-theme)

### Description

When focusing the header cells on https://storybook.saltdesignsystem.com/?path=/story/data-grid-ag-grid-theme--basic-grid a black outline is used for the focus ring.

### Steps to reproduce

1. Visit https://storybook.saltdesignsystem.com/iframe?args=&id=data-grid-ag-grid-theme--basic-grid&viewMode=story
2. Press <kbd>Tab</kbd> until the header is focused
3. Observe a black outline is used

### Expected behavior

The focus ring should be the dotted blue one used by all the other components

### Operating system

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Browser

- [X] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

### Are you a JPMorgan Chase & Co. employee?

- [X] I am an employee of JPMorgan Chase & Co.
